researchapp/services/logging.py

```diff
@@ -0,0 +1,42 @@
+""" Log stuff to ElasticSearch so we can look at it later. """
```

Comment:
That wouldn't be a terrible idea. I imagine that there's actually a lot of overlap between fhir + oauth + logging that these two codebases will share, so some kind of `s4s-fhir-client` module could bundle a lot of that. I'll keep that in mind as I go, and put together a module once a bit more shared functionality has been defined.